torch/_tensor.py

```diff
@@ -81,6 +81,7 @@ def _rebuild_from_type_v2(func, new_type, args, state):
 class Tensor(torch._C.TensorBase):
     _dynamo_static_input_type: str  # Set in `mark_static_address`
     _dynamo_dynamic_indices: Set[int]  # Set in `mark_dynamic`
+    _is_param: bool  # Set on torch.nn.Parameter instances
```

Comment:
This is a change to a core type in pytorch eager, cc @albanD

Reply:
Note that this syntax doesn't have any runtime effect. The attribute is already being set / read in a number of places; this just makes it explicit for mypy.

Reply:
But the point is that this attribute shouldn't exist for regular Tensors. It is not even used for the plain Tensor class.
So adding this here will make the `__annotations__` wrong as this field should NOT be read on a `Tensor` object.

Reply:
It looks like this code is reading it on Tensor objects:

https://github.com/pytorch/pytorch/blob/79e3833703c421cb9cb9ec2c45751a09abbe6e9b/torch/nn/parameter.py#L6-L10